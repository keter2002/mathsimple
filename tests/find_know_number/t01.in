-1 -e 1e-4 3.1415
