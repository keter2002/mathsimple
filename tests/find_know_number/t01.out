3.1415 = pi
